src/formatDate.js

```diff
@@ -50,7 +50,35 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const resultedDate = [];
+  const separatorFrom = fromFormat[3];
+  const separatorTo = toFormat[3];
+
+  const separatedDate = date.split((separatorFrom));
+  let currentYear = separatedDate[fromFormat.indexOf('YY')]
+    || separatedDate[fromFormat.indexOf('YYYY')];
+
+  resultedDate[toFormat.indexOf('MM')]
+    = separatedDate[fromFormat.indexOf('MM')];
+
+  resultedDate[toFormat.indexOf('DD')]
+    = separatedDate[fromFormat.indexOf('DD')];
```

Comment:
Better do it in the right order from the beginning using loop.

Reply:
done